project_euler/problem_074/sol2.py

```diff
@@ -57,7 +65,8 @@ def factorial_sum(a: int) -> int:
     >>> factorial_sum(69)
     363600
     """
-
+    if a in FACTORIAL_SUM_CACHE.keys():
```

Comment:
```suggestion
    if a in FACTORIAL_SUM_CACHE:
```
No need of that. Apply this logic in the above function as well.

Reply:
I personally prefer explicit over implicit, but I agree it's more readable and pretty like this